### PR TITLE
Fix the added document addresses and modified text styles

### PR DESCRIPTION
**Pull request by henwasaia**
added document addresses and modified text styles

### CODE REVIEW

**Comment by henwasaia** [comment]
To solve the problem ：Added document address and removed rainbond related identifier
solution ：write code
test results : Added document addresses and completed the replacement of RainBond related characters

File: src/components/OuterCustomForm/index.js [hunk 1]
```diff
@@ -338,15 +318,16 @@ export default class Index extends PureComponent {
...
-                      <a
-                        href={`${rainbondUtil.documentPlatform_url(
-                          rainbondInfo
-                        )}docs/component-create/thirdparty-service/thirdparty-create`}
-                        target="_blank"
-                        style={{ color: '#fff' }}
-                      >
-                        点击阅读文档
-                      </a>
+                      (platform_url && (
```
Comment (barnettZQG) [inline src/components/OuterCustomForm/index.js:321]: You should click on the icon to jump. Tooltip is just a reminder.

File: src/components/OuterCustomForm/index.js [hunk 2]
```diff
@@ -443,15 +424,16 @@ export default class Index extends PureComponent {
...
-                          <a
-                            href={`${rainbondUtil.documentPlatform_url(
-                              rainbondInfo
-                            )}docs/component-create/thirdparty-service/thirdparty-create`}
-                            target="_blank"
-                            style={{ color: '#fff' }}
-                          >
-                            点击阅读文档
-                          </a>
+                          (platform_url && (
```
Comment (barnettZQG) [inline src/components/OuterCustomForm/index.js:427]: You should click on the icon to jump. Tooltip is just a reminder.

File: src/pages/AddCluster/index.js [hunk 3]
```diff
@@ -625,17 +621,17 @@ export default class EnterpriseClusters extends PureComponent {
                 <Col span={8}>
```
Comment (barnettZQG) [inline src/pages/AddCluster/index.js:621]: The scan module should not be shown in the enterprise version.

File: src/pages/Component/expansion.js [hunk 4]
```diff
@@ -825,7 +825,7 @@ export default class Index extends PureComponent {
                         style={{ background: '#F9FAFC', textAlign: 'center' }}
                       />
                       <div className={styles.remindDesc}>
-                        查询详细的组件实例信息，请复制查询命令到Rainbond管理节点查询
+                        查询详细的组件实例信息，请复制查询命令到平台管理节点查询
```
Comment (barnettZQG) [inline src/pages/Component/expansion.js:828]: "请复制查询命令到平台管理节点查询" > "请复制查询命令到集群管理节点查询"

File: src/pages/Component/index.js [hunk 5]
```diff
@@ -1450,7 +1449,6 @@ class Main extends PureComponent {
             onCancel={this.hideMoveGroup}
           />
         )}
-        <ManageAppGuide />
```
Comment (barnettZQG) [inline src/pages/Component/index.js:1453]: Why is this module deleted?

File: src/pages/Component/setting/auto-deploy.js [hunk 6]
```diff
@@ -281,15 +281,16 @@ export default class AutoDeploy extends PureComponent {
...
-                      <a
-                        href={`${rainbondUtil.documentPlatform_url(
-                          rainbondInfo
-                        )}docs/user-manual/component-dev/auto_build/#基于源代码操作流程`}
-                        target="_blank"
-                        style={{ color: '#fff' }}
-                      >
-                        点击阅读文档
-                      </a>
+                      (platform_url && (
```
Comment (barnettZQG) [inline src/pages/Component/setting/auto-deploy.js:284]: Same as above.

File: src/pages/Component/setting/auto-deploy.js [hunk 7]
```diff
@@ -498,16 +500,17 @@ export default class AutoDeploy extends PureComponent {
...
-                        href={`${rainbondUtil.documentPlatform_url(
-                          rainbondInfo
-                        )}docs/user-manual/component-dev/auto_build/#基于镜像仓库操作流程`}
-                        // eslint-disable-next-line react/jsx-no-target-blank
-                        target="_blank"
-                        style={{ color: '#fff' }}
-                      >
-                        点击阅读文档
-                      </a>
+                      (platform_url && (
```
Comment (barnettZQG) [inline src/pages/Component/setting/auto-deploy.js:503]: Same as above.

File: src/pages/EnterpriseShared/AppExporter.js [hunk 8]
```diff
@@ -63,7 +63,7 @@ export default class AppExporter extends PureComponent {
     return (
       <DescriptionList
         size="large"
-        title="RainbondApp规范(Rainbond平台可用)"
+        title="平台App规范(平台可用)"
```
Comment (barnettZQG) [inline src/pages/EnterpriseShared/AppExporter.js:66]: 平台App规范(平台可用) =》 应用模型规范

File: src/pages/EnterpriseShared/Details.js [hunk 9]
```diff
@@ -556,7 +556,7 @@ export default class Main extends PureComponent {
         )}
         <PageHeaderLayout
           title="应用市场管理"
-          content="应用模型是指模型化、标准化的应用制品包，是企业数字资产的应用化产物，可以通过标准的方式安装到任何Rainbond平台或其他支持的云原生平台"
+          content="应用模型是指模型化、标准化的应用制品包，是企业数字资产的应用化产物，可以通过标准的方式安装到任何平台或其他支持的云原生平台"
```
Comment (barnettZQG) [inline src/pages/EnterpriseShared/Details.js:559]: 修改为：“应用市场支持Rainstore应用商店和Helm应用商店的对接和管理”

File: src/pages/EnterpriseShared/index.js [hunk 10]
```diff
@@ -1122,7 +1122,7 @@ export default class EnterpriseShared extends PureComponent {
     return (
       <PageHeaderLayout
         title="应用市场管理"
-        content="应用模型是指模型化、标准化的应用制品包，是企业数字资产的应用化产物，可以通过标准的方式安装到任何Rainbond平台或其他支持的云原生平台"
+        content="应用模型是指模型化、标准化的应用制品包，是企业数字资产的应用化产物，可以通过标准的方式安装到任何平台或其他支持的云原生平台"
```
Comment (barnettZQG) [inline src/pages/EnterpriseShared/index.js:1125]: Same as above.

File: src/pages/User/registerComponent.js [hunk 11]
```diff
@@ -337,7 +337,7 @@ export default class RegisterComponent extends Component {
               span={24}
               style={{ fontSize: 12, marginTop: -12, color: '#666666' }}
             >
-              请注意：注册使用即同意 Rainbond 发行版用户许可协议。
+              请注意：注册使用即同意平台发行版用户许可协议。
```
Comment (barnettZQG) [inline src/pages/User/registerComponent.js:340]: 注册使用即同意产品发行版用户许可协议。

**Review by barnettZQG — CHANGES_REQUESTED** [review]

**Comment by barnettZQG** [comment]
PR to the master branch.

File: config/config.js [hunk 12]
```diff
@@ -47,11 +47,11 @@ export default {
   routes: routerConfig,
   proxy: {
     '/console': {
-      target: 'http://127.0.0.1:8000',
+      target: 'http://4000.gr7c60a0.2c9v614j.17f4cc.grapps.cn',
```
Comment (barnettZQG) [inline config/config.js:50]: Custom configurations do not commit.

File: src/pages/AddCluster/index.js [hunk 13]
```diff
@@ -622,20 +620,23 @@ export default class EnterpriseClusters extends PureComponent {
...
-                  <p style={{ marginTop: '16px', textAlign: 'center' }}>
-                    <img
-                      alt="扫码加入Rainbond社区钉钉群"
-                      style={{ width: '100%' }}
-                      title="扫码加入Rainbond社区钉钉群"
-                      src="https://www.rainbond.com/images/dingding-group.jpeg"
-                    />
-                  </p>
-                </Col>
+                {/* 企业版的不需要显示扫描模块 */}
```
Comment (barnettZQG) [inline src/pages/AddCluster/index.js:623]: 下面的文字也需要包括在控制域里面啊。

**Review by barnettZQG — CHANGES_REQUESTED** [review]

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**